projects/mercury/src/file/FileBrowser.js

```diff
@@ -275,7 +275,7 @@ export const FileBrowser = (props: FileBrowserProperties) => {
                 selectedPaths={selection.selected}
                 files={files}
                 openedPath={openedPath}
-                openedPathEntityType={openedPathEntityType}
+                openedPathType={openedPathEntityType}
```

Comment:
```suggestion
                openedPathType={openedPathType}
```

change them both?

Reply:
will do, here is where I realized that I need some more refactoring to do, so I left it for later (this property will be removed anyway).